src/modm/platform/core/cortex/ram.ld.in

```diff
@@ -20,12 +14,12 @@ SECTIONS
 
 {{ linker.section_rom("FLASH") }}
 
-{{ linker.section_stack(cont_ram_regions[0].cont_name|upper, "__stack_offset" if vector_table_location == "ram" else None) }}
-
 %% if vector_table_location == "ram"
 {{ linker.section_vector_ram(cont_ram_regions[0].cont_name|upper, table_copy) }}
 %% endif
 
+{{ linker.section_stack(cont_ram_regions[0].cont_name|upper) }}
+
```

Comment:
Don't do this, any stack overflow will overwrite the vector table. Now, any stack overflow will cause a BusError Fault, which is not great either, but at least less heisenbuggy.

Rather use a new linkerscript with this behavior (and document it) placed in `core/stm32` with the other vendor specific scripts.